Update release-notes-8.6.asciidoc (#2642)
elastic/observability-docs: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
====

[[known-issue-issue-2103]]
.Installation of {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the installer
[%collapsible]
====
*Details* +
This issue occurs on MacOS Ventura when Full Disk Access is not granted to the application that runs the installation command.
This could be either a Terminal or any custom package that a user has built to distribute {agent}.

*Impact* +

> {agent} will fail to install and produce "Error: failed to fix permissions: chown elastic-agent.app: operation not permitted" message.

Author: @dedemorton yes there is a workaround. I added the workaround also.

Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
====

[[known-issue-issue-2103]]
.Installation of {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the installer
[%collapsible]
====
*Details* +
This issue occurs on MacOS Ventura when Full Disk Access is not granted to the application that runs the installation command.
This could be either a Terminal or any custom package that a user has built to distribute {agent}.

*Impact* +
{agent} will fail to install and produce "Error: failed to fix permissions: chown elastic-agent.app: operation not permitted" message.

> Ensure that the application (eg. Terminal, or a custom package) used to install {agent} has Full Disk Access before running: sudo ./elastic-agent install

Reviewer: ```suggestion
Ensure that the application used to install {agent} (for example, the Terminal or custom package) has Full Disk Access before running `sudo ./elastic-agent install`.
```
